echopype/utils/io.py

```diff
@@ -32,14 +34,59 @@ def get_files_from_dir(folder):
     return [f for f in os.listdir(folder) if os.path.splitext(f)[1] in valid_ext]
 
 
+def get_zarr_compression(var: xr.Variable, compression_settings: dict) -> dict:
+    """Returns the proper zarr compressor for a given variable type"""
+
+    if np.issubdtype(var.dtype, np.floating):
+        return compression_settings["float"]
+    elif np.issubdtype(var.dtype, np.integer):
+        return compression_settings["int"]
+    elif np.issubdtype(var.dtype, np.str_):
+        return compression_settings["string"]
+    elif np.issubdtype(var.dtype, np.datetime64):
+        return compression_settings["time"]
+    else:
+        raise NotImplementedError(f"Zarr Encoding for dtype = {var.dtype} has not been set!")
+
+
+def set_zarr_encodings(ds: xr.Dataset, compression_settings: dict):
+    """
+    Sets all variable encodings based on zarr default values
+    """
+
+    # create zarr specific encoding
+    encoding = dict()
+    for name, val in ds.variables.items():
+
+        val_encoding = val.encoding
+        val_encoding.update(get_zarr_compression(val, compression_settings))
+        encoding[name] = val_encoding
+
+    return encoding
+
+
 def save_file(ds, path, mode, engine, group=None, compression_settings=None):
     """Saves a dataset to netcdf or zarr depending on the engine
     If ``compression_settings`` are set, compress all variables with those settings"""
```

Comment:
```suggestion
    """
    Saves a dataset to netcdf or zarr depending on the engine
    If ``compression_settings`` are set, compress all variables with those settings
    """
```
Just so that the docstring format is uniform...